### PR TITLE
Fixes immersion ruining clothesmate line

### PR DESCRIPTION
**Pull request by gy1ta23**
## About The Pull Request

Changes "Look at all this free swag!" clothesmate voice line to "Look at all this not free swag!" because we have to pay for it now

## Why It's Good For The Game
This change will maybe, probably, hopefully greatly increase immersion. And I used desktop this time, Oranges.

## Changelog
:cl:
tweak: updates clothesmate line 
/:cl:

### CODE REVIEW

**Comment by Tlaltecuhtli** [comment]
a

**Comment by subject217** [comment]
don't pr from your master, just make a branch, it's not hard

File: code/modules/vending/clothesmate.dm [hunk 1]
```diff
@@ -4,7 +4,7 @@
 	desc = "A vending machine for clothing."
 	icon_state = "clothes"
 	icon_deny = "clothes-deny"
-	product_slogans = "Dress for success!;Prepare to look swagalicious!;Look at all this free swag!;Why leave style up to fate? Use the ClothesMate!"
+	product_slogans = "Dress for success!;Prepare to look swagalicious!;Look at all this not free swag!;Why leave style up to fate? Use the ClothesMate!"
```
Comment (81Denton) [inline code/modules/vending/clothesmate.dm]:
```suggestion
	product_slogans = "Dress for success!;Prepare to look swagalicious!;Look at all this swag!;Why leave style up to fate? Use the ClothesMate!"
```

Comment (81Denton) [reply code/modules/vending/clothesmate.dm]: Tbh just change it to that

Comment (ExcessiveUseOfCobblestone) [reply code/modules/vending/clothesmate.dm]: I lol'd

**Comment by gy1ta23** [comment]
well you see it's from my master because the last time I tried to make a PR, it got shut down because I used the webeditor. So I had to do some bullshit with desktop instead of actually figuring out how it works

File: code/modules/vending/clothesmate.dm (hunk 1 above)
Comment (gy1ta23) [reply code/modules/vending/clothesmate.dm]: I think the not free really adds to the immersion though, really makes it a better line in general

Comment (ExcessiveUseOfCobblestone) [reply code/modules/vending/clothesmate.dm]: I lol'd _x2_

**Comment by ExcessiveUseOfCobblestone** [comment]
I would totally webbit a change like this tbh.

**Comment by JJRcop** [comment]
If you're using **GitHub Desktop**, take a look at this article:

**https://help.github.com/desktop/guides/contributing-to-projects/making-changes-in-a-branch/**

--------------------------------------------------------------------------------------------------------------------------------

If you're using **git gui**, you can use the Branch menu to manage branches.

screenshot_20190206_130230

-------------------------------------------------------------------------------------------------------------------------------

If you're using **_git_ on the command line**, there's the [Branching chapter](https://git-scm.com/book/en/v1/Git-Branching) of the Pro Git book you can read. The [Git Branching - Branches in a Nutshell](https://git-scm.com/book/en/v2/Git-Branching-Branches-in-a-Nutshell) section is enough, but keep reading if you'd like to know more or don't understand.

The gittutorial man page also has a section on branching on the command line.

**https://git-scm.com/book/en/v2/Git-Branching-Branches-in-a-Nutshell**

**https://git-scm.com/docs/gittutorial#_managing_branches**

**Comment by pubby** [comment]
Joke's funny now, but it won't be in a week.

**Comment by gy1ta23** [comment]
It's not a joke, it's a legitimate feature that I believe will greatly improve the game

**Comment by wesoda25** [comment]
Thank you. Lately I just haven’t been feeling immersed. This will change that, I’m sure.

**Review by ShizCalev — CHANGES_REQUESTED** [review]
what @81Denton said.

**Comment by gy1ta23** [comment]
done

**Pull request merged**